.gitignore

```diff
@@ -1,3 +1,4 @@
+/.php_cs.cache
```

Comment:
just for future Docker evolutions, is that the only cache needed? Is it a file or a folder?

Reply:
Just this file. In the next version you can also specify the location. I envisaged mounting the file to persist the cache between builds.